tests/helpers/Messages/createMessage.test.js

```diff
@@ -0,0 +1,32 @@
+const createMessage = require("../../../helpers/Messages/createMessage");
+const createUser = require("../../../helpers/Users/createUser");
+const { createTables, wipeDBTables } = require("../../../db/databaseHelpers");
+
+beforeAll(async() =>{
+  await createTables();
+  await createUser("1", "hel", "password", "student");
+  await createUser("2", "jka", "pasd", "tutor");
+});
+afterAll(async() =>{
+  await wipeDBTables();
+});
+describe("creating a message", () =>{
+  it("should return true", async() =>{
```

Comment:
Instead of saying it should return true, you should give the `it` description some context as to what the actual behavior is doing.

Better: `it("saves the message in the database", ...`

Reply:
a54ce19c9dd3545dade37ef2adffa4695b12b370